components/ServiceItem.tsx

```diff
@@ -12,19 +12,18 @@ import {
   Button,
   Heading,
   HStack,
-  Link,
   LinkBox,
   LinkOverlay,
   Stack,
   Text,
 } from '@chakra-ui/react'
-import { Address, Service } from 'models'
+import { AddressWithLabel, Resource } from 'models'
```

Comment:
Addresses in the new Airtable schema aren't in a separate table with names attached so created a new type that links addresses with labels.